docs/user/running-cincinnati.md

```diff
@@ -16,6 +16,34 @@ The Cincinnati deploymentconfig is available [here](../../dist/openshift/cincinn
 oc new-project cincinnati
 ```
 
+#### Create secret for container registry
+
+If Cincinnati needs credentials to access a container registry to fetch the release payload information , you need to create a secret with name `cincinnati-credentials`. Make sure the auth for the container registry is present in the config.json file.
```

Comment:
nit: ` , ` -> `, ` (removing the space before the comma).

Also, is the name hard-coded?  I'd have expected this repository to not care about particular secrets at all.  Is the deployment template expecting this particular secret name?  Do we want to start rolling back dependency on the deployment template and start using the new operator?  How do you feed this information into the operator?

Reply:
> Is the deployment template expecting this particular secret name?

[Yes](https://github.com/openshift/cincinnati/blob/master/dist/openshift/cincinnati.yaml#L160-L162). The secret may also contain other credentials - for github etc.

Reply:
fixed in https://github.com/openshift/cincinnati/pull/296/commits/57b9b01ed74d0a80299ea81a2d01047f1b6b6bc8